### PR TITLE
Update Baro debug

### PR DESCRIPTION
**Pull request by haslinghuis**
Updates Baro debug changed in https://github.com/betaflight/betaflight/pull/11911

### CODE REVIEW

**Comment by KarateBrot** [comment]
Dang, that was fast 🚀

File: js/flightlog_fields_presenter.js
Comment (KarateBrot) [inline js/flightlog_fields_presenter.js:999]:
```suggestion
                        case 'debug[1]':
                            return `${value.toFixed(0)} hPa`;
                        case 'debug[2]':
                            return `${(value / 100).toFixed(2)} °C`;
                        case 'debug[3]':
                            return `${(value / 100).toFixed(2)} m`;
                        default:
                            return `${(value).toFixed(0)}`;
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

**Review by KarateBrot — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
